docs.md

```diff
@@ -1,23 +1,396 @@
+   ```json
+   {
+      "command": "input_prop_equals",
+      "properties": {
+         "input_property": "company",
+         "value": "Geobeyond srl"
+      }
+   }
+   ```
+
+ ### -  Allow access to a particular path: <br />
+ This command is to allow access to a particular path e.g `/v1/collections/obs` if the property of the request equals certain value<br />
+
+   `allow {
+  input.request_path == ["v1", "collections", "lakes", ""] input.groupname == "admin"
+}` <br /> 
+   and the rule object is:
+   ```json
+   [
+      {
+         "command": "input_prop_equals",
+         "properties": {
+            "input_property": "request_path",
+            "value": ["v1", "collections", "lakes"],
+         },
+      },
+       {
+         "command": "input_prop_equals",
+         "properties": {
+            "input_property": "request_path",
+            "value": "admin",
+         },
+      }
+   ]
+   ```
+   ## 2. Input_props_in
+   This command is to check if a particular property on the request object is in a list of values from the database. <br />
+
+   `allow {
+        input.company == data.items[i].name
+   }` <br /> 
+   and the rule object is:
+   ```json
+   {
+      "command": "input_prop_in",
+      "properties": {
+         "input_property": "company",
+         "datasource_name": "items",
+         "datasource_loop_variable": "name"
+      }
+}
+   ```
+   ## 3. Input_props_in_as
+   This command is to check if two properties on the request object is present on the same object in the database <br />
+
+
+   `allow {
+         some i \n
+         data.items[i].name == input.preferred_username \n
+         data.items[i].everyone == input.groupname 
+   }` <br /> 
+   and the rule object is:
+   ```json
+ {
+      "command": "input_prop_in_as",
+      "properties": {
+         "datasource_name": "items",
+         "datasource_loop_variables": ["name", "everyone"],
+         "input_properties": ["preferred_username", "groupname"]
+      }
+ }
+   ```
+
+
+## 4. allow_full_access
+   This command is to allow full access to the resource, if the property on the request object has a particular value<br />
+
+
+   `allow {
+  input.groupname == "EDITOR_ATAC"
+}
+   }` <br /> 
+   and the rule object is:
+   ```json
+   [
+      {
+         "command": "allow_full_access",
+         "properties": {
+            "input_property": "groupname",
+            "value": "EDITOR_ATAC",
+         },
+      }
+   ]
+   ```
+
+
+   API CRUD Operations
+   ============
+
+   The API is designed to support all CRUD operations that makes interaction with the policies easy. The following routes are currently defined:
+
+ ###  - POST `/policies/`  🗝
+Using the JSON defined from the frontend, this route is used to build a new policy. The response will be a REGO file written and pushed to github as a newly established remote repository, with the request body conforming to a specified syntax described by the pydantic model `Policy`. <br />
+```py3
```

Comment:
Instead:

The `/policies` route receives a JSON request body comprising the rules needed to create a new policy. The policy upon a successful operation is created and the defined GitHub repository is updated with the policy file of _.rego_ format.

Anyone can choose to use the API and not necessarily a frontend..